Plan/common/src/main/resources/assets/plan/web/js/graphs.js

```diff
@@ -148,32 +149,36 @@ function dayByDay(id, series) {
     }));
 }
 
-function onlineActivityCalendar(id, events, firstDay) {
-    $(id).fullCalendar({
+function onlineActivityCalendar(id, event_data, firstDay) {
+    window.calendars.online_activity = new FullCalendar.Calendar(document.querySelector(id), {
+        themeSystem: 'bootstrap',
         eventColor: '#2196F3',
         firstDay: firstDay,
+        initialView: 'dayGridMonth',
 
-        eventRender: function (eventObj, $el) {
-            $el.popover({
-                content: eventObj.title,
+        eventDidMount: function (info) {
+            $(info.el).popover({
+                content: info.event.title,
                 trigger: 'hover',
                 placement: 'top',
                 container: 'body'
             });
         },
 
-        events: events,
+        events: function (fetchInfo, successCallback, failureCallback) {
+            successCallback(event_data)
+        },
 
         height: 800,
         contentHeight: 795,
-        header: {
+        headerToolbar: {
             left: 'title',
             center: '',
-            right: 'month prev,next'
+            right: 'dayGridMonth today prev,next'
```

Comment:
Answering the question, The month button might be unnecessary in this case. (unless you choose the week view workaround)